Solutions/Microsoft Entra ID/Analytic Rules/ExplicitMFADeny.yaml

```diff
@@ -49,14 +49,20 @@ entityMappings:
   - entityType: IP
     fieldMappings:
       - identifier: Address
-        columnName: IPAddress
+        columnName: PublicIP
   - entityType: AzureResource
     fieldMappings:
-      - identifier: ResourceID
-        columnName: ResourceID
-  - entityType: URL
+      - identifier: ResourceId
+        columnName: ResourceId
+  - entityType: CloudApplication
```

Comment:
can you also add  identifier: AppId and Fieldmapping for CloudApplication entity.